winui/Kanban/swimlane.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: Swimlane support in WinUI Kanban control | Syncfusion
+description: Learn here all about the swimlane support and its customization in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Swim lanes in WinUI Kanban Control
+
+The Swim lanes organize the Kanban board horizontally, allowing you to categorize cards by key attributes such as assignee, project, or task type.
+
+By default, swimlanes are categorized based on the `Assignee` property in the `KanbanModel` class. However, you can customize the categorization by mapping the `SwimlaneKey` property to the appropriate property name in `KanbanModel`.
```

Comment:
By default, swimlanes are categorized based on the `Assignee` values in the `KanbanModel` class. You can also define category by mapping `SwimlaneKey` to appropriate property name in the defined kanban model.

Reply:
Changed.